RULES.md

```diff
@@ -429,3 +430,9 @@ At least one of the fields `is_producer`, `is_operator`, or `is_authority` shoul
 
 #### References:
 * [attributions.txt specification](https://gtfs.org/reference/static#attributionstxt)
+
+<a name="StopTimeTimepointWithoutTimeNotice"/>
+
+### StopTimeTimepointWithoutTimeNotice
+
+Timepoints from GTFS file `stop_times.txt` should defined values for both `stop_times.arrival_time` and `stop_times.departure_time` fields. 
```

Comment:
The description should include "If `timepoint` is set to 1..."

There is another case where the timepoint column is omitted, and arrival and departure times are omitted for non-timepoints and all other records with times are assumed to be timepoints - this was the original definition for timepoint before the `timepoint` field was introduced. And we're not addressing this case here.

Reply:
> There is another case where the timepoint column is omitted, and arrival and departure times are omitted for non-timepoints and all other records with times are assumed to be timepoints - this was the original definition for timepoint before the timepoint field was introduced. And we're not addressing this case here.

Opening an issue to keep track of this.